testdata/kata/webhook/example-fedora.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
```
$ cat testdata/kata/example-fedora-kata.yaml
apiVersion: v1
kind: Pod
metadata:
  name: example-fedora-kata
  labels:
    app: example-fedora-kata-app
spec:
  containers:
    - name: example-fedora
      image: fedora:30
      ports:
        - containerPort: 8080
      command: ["python3"]
      args: [ "-m", "http.server", "8080"]
  runtimeClassName: kata
```

Reply:
We already have a file with the same content there. Perhaps we should fix the filename in the scenario instead of create a new file with the exact content.

Reply:
@liangxia those two files are different if you look carefully or diff them
```
$ diff example-fedora-kata.yaml webhook/example-fedora.yaml
4c4
<   name: example-fedora-kata
---
>   name: example-fedora
6c6
<     app: example-fedora-kata-app
---
>     app: example-fedora-app
15d14
<   runtimeClassName: kata
```